src/rtos/FreeRTOS.c

```diff
@@ -33,26 +33,13 @@
 #include "target/cortex_m.h"
 
 #define FREERTOS_MAX_PRIORITIES	63
-
-#define FreeRTOS_STRUCT(int_type, ptr_type, list_prev_offset)
-
-/* FIXME: none of the _width parameters are actually observed properly!
- * you WILL need to edit more if you actually attempt to target a 8/16/64
- * bit target!
- */
+#define FREERTOS_THREAD_NAME_STR_SIZE 200
 
 struct FreeRTOS_params {
 	const char *target_name;
-	const unsigned char thread_count_width;
```

Comment:
I'm curious of the rationale for moving these data members to struct FreeRTOS; is it because those fields get assigned by querying RTOS data structures at runtime?  (As opposed to being parameters provided in OpenOCD init scripts etc?

Reply:
That's exactly why. Specifically, FreeRTOS_params is defined as const, and it didn't seem right to make it writable.